src/blip/Message.cc

```diff
@@ -343,7 +343,7 @@ namespace litecore { namespace blip {
 
     Error MessageIn::getError() const {
         if (!isError())
-            return {};
+            return {nullslice, 0, nullslice};
```

Comment:
Actually there may be another way around it by defining a default constructor.  Here is the full error message:

> error C3852: 'litecore::blip::Error::code' having type 'const int': aggregate initialization could not initialize this member.  note: const members cannot be default initialized unless their type has a user defined default constructor